var/spack/repos/builtin/packages/bowtie/package.py

```diff
@@ -33,7 +34,7 @@ class Bowtie(MakefilePackage):
     # builds with %gcc@5.5.5 and fails to build with %gcc@8.2.0.  I'm
     # not sure whether or not it works with other versions in the
     # interval.
-    conflicts('%gcc@8:', when='@1.2.2:')
+    conflicts('%gcc@8:', when='@1.2.2')
```

Comment:
Can you check if `bowtie` has issues with older versions too, so that we'll have a more reliable conflict statement? In case update the comment above.